css/At.scss

```diff
@@ -150,8 +150,12 @@
 		}
 
 		.atwho-cur {
-			background: var(--color-primary);
+			background: var(--color-primary-element);
```

Comment:
If we use --color-primary-light, then we don’t have to change the text color, and the status is also distinguished for the person who is selected. That way it also looks consistent with the global style as per Vue components and https://github.com/nextcloud/server/pull/22115

Reply:
I tried it together with Marco, it is much worse with certain combinations.

Reply:
nevermind, I used element-light, instead of only light

Reply:
fixed